src/wazuh_db/helpers/wdb_agents_helpers.c

```diff
@@ -117,16 +133,18 @@ int wdb_agents_vuln_cve_clear(int id,
         wdbc_close(&aux_sock);
     }
 
+    cJSON_Delete(data_in);
+    os_free(data_in_str);
     os_free(wdbquery);
     os_free(wdboutput);
 
     return result;
 }
 
-int wdb_agents_vuln_cve_update_status(int id,
-                               const char *old_status,
-                               const char *new_status,
-                               int *sock) {
+int wdb_agents_vuln_cve_remove_entry(int id,
```

Comment:
Shouldn't be wdb_agents_vuln_cves_remove_entry?

Reply:
Fixed!